src/main/java/seedu/address/model/Statistics.java

```diff
@@ -32,31 +35,34 @@
     private Integer currentYear;
     private Integer currentMonth;
 
-    //@@author 500poundbear
     public Statistics (ObservableList<ReadOnlyPerson> list, int currentMonth, int currentYear) {
-
-        this.currentYear = currentYear;
-        this.currentMonth = currentMonth;
-
-        this.personList = list;
-
-        tabulateTotalNumberOfPeople();
+        setStatisticAttributes(list, currentMonth, currentYear);
         tabulateSocialMediaUsage();
     }
 
+    /**
+     * Returns Array for each month in past displayYears with a count of new persons added in that month
+     * Current month is also included
+     *
+     * @param displayYears The number of years to be displayed
```

Comment:
I think maybe can be consistent with the commenting, other methods dont have @param and @return in their comments